addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -377,6 +382,7 @@ PortalChatter.include({
     _onChangeRatingDomain: function () {
         var domain = [];
         if (this.get('rating_value')) {
+            // TODO dead code: remove in master
             domain = [['rating_value', '=', this.get('rating_value')]];
```

Comment:
1. There is a lot of other dead code surrounding this domain removal. I am removing it all in master (as needed as I am adapting the code surrounding this). I already did it in my master branch but will move it into the forward-port of this one once it reaches master.
2. It may not be dead code for customization: there is no value in removing it in stable, only risks (small of course but still > 0).